src/pages/Spacing.vue

```diff
@@ -0,0 +1,304 @@
+  background-image: linear-gradient(135deg,#3b82f680 10%,transparent 0,transparent 50%,#3b82f680 0,#3b82f680 60%,transparent 0,transparent);
+  background-size: 7.07px 7.07px;
+}
+.bg-stripes-fuchsia {
+  background-color: #e879f91a;
+  background-image: linear-gradient(135deg,#d946ef80 10%,transparent 0,transparent 50%,#d946ef80 0,#d946ef80 60%,transparent 0,transparent);
+  background-size: 7.07px 7.07px;
+}
+
+/* Support, yay or nay? They are nice */
+/* Same effect as -m-4 on parent and m-4 on children */ 
+.space-x-4 > * + * {
+  margin-left: 1rem; 
+}
+
+</style>
+<template>
+  <h1>Basic usage</h1>
+  <container>
+    <box class="flex flex-wrap items-start justify-center text-white text-sm font-bold leading-6 -mx-5">
+      <div class="flex items-start">
+        <div class="flex-none px-5">
+          <div class="bg-purple-500 shadow-lg rounded-lg overflow-hidden">
+            <div striped class="h-6 bg-stripes-white striped rounded-t-lg"></div>
+            <div class="p-4">pt-6</div>
+          </div>
+        </div>
+        <div class="flex-none px-5 pt-6">
+          <div class="flex bg-purple-500 shadow-lg rounded-lg overflow-hidden">
+            <div class="flex-none p-4">pr-4</div>
+            <div class="flex-none w-4 bg-stripes-white"></div>
+          </div>
+        </div>
+      </div>
+      <div class="flex items-start">
+        <div class="flex-none px-5 pt-6">
+          <div class="bg-purple-500 shadow-lg rounded-lg overflow-hidden">
+            <div class="p-4">pb-8</div>
+            <div class="h-8 bg-stripes-white"></div>
+          </div>
+        </div>
+        <div class="flex-none flex px-5 pt-6">
+          <div class="flex bg-purple-500 shadow-lg rounded-lg overflow-hidden">
+            <div class="flex-none w-2 bg-stripes-white"></div>
+            <div class="flex-none p-4">pl-2</div>
+          </div>
+        </div>
+      </div>
+    </box>
+  </container>
+  <h2>Code example</h2>
+  <container>
+    <!-- Placeholder for a syntax-higlighter -->
+    <syntax-highlighter>
+      <!--
+        <div class="pt-6 ...">pt-6</div>
+        <div class="pr-4 ...">pr-4</div>
+        <div class="pb-8 ...">pb-8</div>
+        <div class="pl-2 ...">pl-2</div>
+      -->
+    </syntax-highlighter>
+  </container>
+  <h1>Horizontal Padding</h1>
+  <container>
+    <box class="relative rounded-xl overflow-auto p-8">
+      <div class="flex justify-center font-mono text-white text-sm font-bold leading-6">
+        <div class="bg-indigo-500 rounded-lg shadow-lg overflow-hidden flex">
+          <div class="w-8 bg-stripes-white"></div>
+          <div class="p-4">px-8</div>
+          <div class="w-8 bg-stripes-white"></div>
+        </div>
+      </div>
+    </box>
+  </container>
+
+  <h2>Code example</h2>
+  <container>
+    <syntax-highlighter>
+      <!--
+        <div class="px-8 ...">px-8</div>
+      -->
+    </syntax-highlighter>
+  </container> 
+  
+
+  <h1>Vertical padding</h1>
```

Comment:
There's multiple h1's on this page, might not be an issue, but that should probably be cleaned up at some point

Reply:
Yea your right about that :) This is not the actual documentation though, I think these are going to be moved to theyr actual docpages eventually, where the h levels needs to fit the rest of the dom.

Reply:
If these each went into `section`s isn't that valid semantics?

Reply:
> If these each went into sections isn't that valid semantics?

yeah, I believe that would make them valid semantically